src/viking.js

```diff
@@ -1,11 +1,82 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor(health, strength) {
+        this.health = health;
+        this.strength =  strength;
+    }
+    attack() {
+        return this.strength;
+    }
+    receiveDamage(damage) {
+        let currentHealth = this.health - damage;
+    }
+}
+
+
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+    constructor(health, strength, name){
```

Comment:
`name` should be the first parameter in the constructor

```suggestion
    constructor(name,health, strength){
```